tests/unit/views/ont/OntPlates.spec.js

```diff
@@ -59,10 +60,11 @@ describe('OntPlates.vue', () => {
       expect(button.text()).toEqual('Show Plate')
     })
 
-    it('has a OntPlate component on button click', () => {
+    it('has a OntPlate component on button click', async () => {
       button = wrapper.find('#details-btn-1')
       button.trigger('click')
-      expect(wrapper.contains(OntPlate)).toBe(true)
+      await flushPromises()
```

Comment:
This shouldnt be needed, maybe the await can be moved up a line on `await button.trigger('click')`